zerocopy-derive/src/lib.rs

```diff
@@ -413,7 +413,7 @@ fn derive_from_zeros_struct(ast: &DeriveInput, strct: &DataStruct) -> proc_macro
 // - one of the variants has a discriminant of `0`
 
 fn derive_from_zeros_enum(ast: &DeriveInput, enm: &DataEnum) -> proc_macro2::TokenStream {
-    if !enm.is_c_like() {
+    if !enm.is_fieldless() {
         return Error::new_spanned(ast, "only C-like enums can implement FromZeros")
```

Comment:
Can we fix up these error messages too?
```suggestion
        return Error::new_spanned(ast, "only fieldless enums can implement FromZeros")
```